style.css

```diff
@@ -0,0 +1,103 @@
+
+* {
+    margin: 0; /* Убираем отступы */
+    padding: 0;
+}
+
+
+body {
+    height: 100%; /* Высота страницы */
+    background: url(back.jpg); /* Параметры фона */
+    background-size: cover; /* Фон занимает всю доступную площадь */
+}
+p {
+    
+}
+
+
+.mainblock {
+	
+	position: relative;
+    width: 600px; /*задаем ширину блока*/
+  	height:400px; /*задаем высоту блока*/
+  	transform: translate(-50%, -50%);
```

Comment:
make sure to read this https://www.w3schools.com/cssref/css3_pr_transform.asp to better understand how it works